path/common.ts

```diff
@@ -16,6 +16,9 @@ import { SEP } from "./separator.ts";
  *       console.log(p); // "./deno/std/"
  * ```
  */
-export function common(paths: string[], sep: typeof SEP = SEP): string {
+export function common(
+  paths: string[],
+  sep: string = SEPARATOR,
```

Comment:
Sidenote: I see no use case for this `sep` argument. Ditto for the Windows version. Perhaps, it should be deprecated.

Reply:
Probably work for another PR.